src/controllers/base.controller.js

```diff
@@ -34,4 +39,28 @@ module.exports = class BaseController {
       return controllerSubclass.doPost(request, reply, errors)
     }
   }
+
+  static async generateCookie (reply) {
+    // Generate a session token
+    const token = uuid4()
+
+    // Generate a CRM token
+    let authToken
+    try {
+      authToken = await authService.getToken()
+    } catch (err) {
+      console.error(err)
+      return reply.redirect('/error')
```

Comment:
There is now a constant defined for the /error route:

- Constants.Routes.ERROR.path

Reply:
Thanks, I'll get that updated!